librz/core/cmd/cmd_api.c

```diff
@@ -1193,6 +1171,30 @@ static void fill_details(RzCmd *cmd, RzCmdDesc *cd, RzStrBuf *sb, bool use_color
 	}
 }
 
+static char *group_get_help(RzCmd *cmd, RzCmdDesc *cd, bool use_color) {
```

Comment:
Why the move below? If it is not necessary, could you move it back to the original position so that it's clearer what are the differences here?

Reply:
Just to avoid the forward decl. But sure, I can move it back.